src/librustdoc/html/highlight.rs

```diff
@@ -69,6 +69,7 @@ fn write_header(out: &mut Buffer, class: Option<&str>, extra_content: Option<Buf
     } else {
         writeln!(out, "<pre class=\"rust\">");
     }
+    write!(out, "<code>");
```

Comment:
I think this is not ideal because before we have `writeln!`, what about:

```rust
    if let Some(class) = class {
        writeln!(out, "<pre class=\"rust {}\"><code>", class);
    } else {
        writeln!(out, "<pre class=\"rust\"><code>");
    }
```

?

Reply:
I did it in this way because if I also used `writeln!` here, it would generate `<pre>\n<code>\n`, which added a new empty line at the beginning of the code block. I originally wanted to use the code that you have posted, but it seemed like a needless duplication of `<code>`.

Is there something that you don't like with the current code here? I'm not sure what is the problem.

Reply:
I want the backline after the tags. If the backline isn't needed, then better to replace `writeln!` with `write!`.

Reply:
I see. I think that the newline isn't actually even needed, so I'll try to remove it. If it's needed, I'll put it after the tags.